containers/kubernetes/how-to/manage-kubeconfig-with-iam.mdx

```diff
@@ -0,0 +1,66 @@
+---
+meta:
+  title: How to manage Kubeconfig files with IAM
+  description: This page explains how to manage Scaleway Kubeconfig files with IAM
+content:
+  h1: How to manage Kubeconfig files with IAM
+  paragraph: This page explains how to manage Scaleway Kubeconfig files with IAM
+tags: kubernetes kapsule-cluser
+dates:
+  validation: 2024-11-05
+  posted: 2024-11-05
+categories:
+  - kubernetes
+---
+
+
+A Kubeconfig file serves as a repository for essential credentials required to access a Kubernetes cluster and manage deployments.  
+This file contains critical information, including cluster details, authentication data, and context information, allowing users to oversee and configure their Kubernetes clusters.
+
+Scaleway's Identity and Access Management (IAM) is now being used to generate Kubeconfig files. This integration offers several advantages:
+
+- **Improved user administration:**
+   - When removing a cluster user, there is no longer a need to reset the admin token, mitigating security risks.
+- **Increased cluster security:**
+   - Each user has unique credentials, enhancing security.
+   - Audit logs facilitate the identification of specific Kubernetes users.
+- **Enhanced application security:**
+   - Applications within the cluster are authenticated using specific tokens.
+   - This authentication method is particularly useful when a pod needs to interact with the Kubernetes API, as it can authenticate using its associated [Service Account](https://kubernetes.io/docs/concepts/security/service-accounts/).
+
+[Identity and Access Management (IAM)](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#iam) provides control over resource access. IAM policies enable the configuration of permissions for Kubernetes Kapsule clusters at the Project level.
+
+An [IAM policy](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#policy) defines the permissions for users, groups, and applications within an Organization. It consists of a [principal](https://www.scaleway.com/en/docs/identity-and-access-management/iam/concepts/#principal) (the user, group, or application to which it applies) and IAM rules that specify permission sets and their scope.
+
+The [combination of IAM and Kubernetes RBAC (Role-based Acccess Control)](/containers/kubernetes/reference-content/set-iam-permissions-and-implement-rbac/) allows you to define fine-grained access levels for cluster users.
+
+<Message type="important">
+    Entering a deprecation cycle, the former authentication system (using static admin tokens) remains active until the second half of 2024: all your legacy Kubeconfig files keep access to their clusters for the time being. For security purposes, you can still use the ["reset admin token" feature](/containers/kubernetes/how-to/revoke-kubeconfig/), either through our Kubernetes API or Scaleway’s CLI.
```

Comment:
Is it really second half of 2024? Or Q1 2025, as we're mentioning on the console?